src/stream/src/executor/eowc/eowc_gap_fill.rs

```
@@ -103,7 +106,7 @@ impl<S: StateStore> ExecutorInner<S> {
                     "Failed to convert time column to timestamp, got {:?}. Skipping gap fill.",
                     prev_time_scalar
                 );
-                return Ok(filled_rows);
```

It should return an error.